README.md

```diff
@@ -155,6 +155,12 @@ In the event service call any variable coming from the MQTT message can be used
               entity_id: climate.thermostat_1
               temperature: "{{ val | int / 10 }}"
       - obj: "p2b4"
+        properties:
+          "options": >
+            {% for mode in state_attr('climate.thermostat_1','hvac_modes') %}
+            {{ mode }}{{ "\n"|e }}{%- if not loop.last %}
```

Comment:
see if this works:

{{ mode + "\n" | e }}

Reply:
it works.
I just noticed that there's a space before all items except the first one:
![kép](https://user-images.githubusercontent.com/1550668/112353204-d08bba80-8ccb-11eb-8bf1-1d49c74633ae.png)

Reply:
you would need to  erase all spaces in the template

Reply:
Cleared everything, but still.
```
            {%for mode in state_attr('climate.halo_klima','hvac_modes')%}
            {{mode+"\n"|e}}{%-if not loop.last%}{%-endif%}{%-endfor%}
```

Reply:
:( no clue....

Reply:
if it's all in one line it's OK: `{%for mode in state_attr('climate.thermostat_1','hvac_modes')%}{{mode+"\n"|e}}{%-if not loop.last%}{%-endif%}{%-endfor%}`
Updated the patch.